docs/operation/node_management/bootstrap_backup_restore.rst

```diff
@@ -1,30 +1,30 @@
-Bootstrap Node Backup and Restoration Procedure
-===============================================
+Backing Up and Restoring a Bootstrap Node
+=========================================
 
 This section describes how to backup a **MetalK8s** bootstrap node
-and how to restore a bootstrap node from such backup.
+and how to restore a bootstrap node from a backup.
 
-Backup procedure
-****************
+Backing Up a Node
+*****************
 
 A backup file is generated at the end of the bootstrap.
 
-To create a new backup file you can run the following command:
+Create a new backup file by running the following command:
 
 .. code::
 
     /srv/scality/metalk8s-X.X.X/backup.sh
 
-Backup archives are stored in `/var/lib/metalk8s/`.
+Backup archives are stored in ``/var/lib/metalk8s/``.
 
-Restoration procedure
-*********************
+Restoring a Node
+****************
 
 .. warning::
 
-   It is mandatory to have a highly available control plane, with at least
-   3 members in the ``etcd`` cluster (including the failed bootstrap Node),
-   to use the restore script.
+   To use the restore script, it is mandatory to have a highly available
```

Comment:
Using the restore script requires a highly available control plane with at least